lib/Parse/CPAN/Meta.pm

```diff
@@ -155,21 +155,21 @@ __END__
 
     #############################################
     # In your file
-    
+
     ---
     name: My-Distribution
     version: 1.23
     resources:
       homepage: "http://example.com/dist/My-Distribution"
-    
-    
+
+
```

Comment:
Yes "trailing spaces".

I thought this splits verbatim blocks in two. But I had a fresh look at [perlpodspec](http://perldoc.perl.org/perlpodspec.html#Notes-on-Implementing-Pod-Processors) and it says:
> Pod parsers, when processing a series of verbatim paragraphs one after another, should consider them to be one large verbatim paragraph that happens to contain blank lines.

So I'm sorry. I was wrong.